conda/recipes/rapids-build-env/meta.yaml

```
@@ -59,6 +61,7 @@ requirements:
     - cuda-python {{ cuda_python_version }}
     - cudatoolkit ={{ cuda_major }}.*
     - cupy {{ cupy_version }}
+    - cxx-compiler {{ cxx_compiler_version }}
```

Would recommend including the `c-compiler` too

Also the compiler versions are [a bit wonky]( https://github.com/conda-forge/compilers-feedstock/blob/084a965125ed37df62f944c027ca6d64806b35e8/recipe/meta.yaml#L29 ). So would skip pinning these directly

Though we can do this indirectly by pinning `gcc_impl_*`

```suggestion
    - c-compiler
    - cxx-compiler
    - gcc_impl_{{ target_platform }}  {{ gcc_version }}
```